client/lib/analytics/utils/cart-to-ga-purchase.ts

```diff
@@ -11,18 +11,20 @@ export type GaPurchase = {
 	items: GaItem[];
 };
 
-export function jetpackCartToGaPurchase(
+export function cartToGaPurchase(
 	orderId: string,
 	cart: ResponseCart,
 	cartInfo: WpcomJetpackCartInfo
 ): GaPurchase {
+	const value = cartInfo.containsWpcomProducts ? cartInfo.wpcomCostUSD : cartInfo.jetpackCostUSD;
```

Comment:
@sretika -- I think we can remove this conditional now, since we're already checking for `wpcomJetpackCartInfo.containsJetpackProducts` in `recordOrderInJetpackGA` in `record-order.js`. Will simplify it.

Reply:
Hi @gmovr! Just saw that you are updating cart items. Can you please include product type? e.g. product type: wp-bundle, domain, difm. Plan type (personal, premium), frequency (monthly, annual) and amount associated? Also the amount is net (includes discount right). Coupon codes will be awesome too.

Reply:
@debbiengWP - Yes, I think we should do this in a follow-up. I need to research more how we can use this meta data -- but I definitely think this is a good idea.